hazelcast/src/main/java/com/hazelcast/partition/impl/PartitionStateOperation.java

```diff
@@ -46,6 +48,11 @@ public PartitionStateOperation(PartitionRuntimeState partitionState, boolean syn
 
     @Override
     public void run() {
+        NodeEngineImpl nodeEngine = (NodeEngineImpl) getNodeEngine();
+        if (nodeEngine.getNode().getState() == NodeState.PASSIVE) {
+            getLogger().warning("Partition table received before startup is not completed. Caller: " + getCallerAddress());
```

Comment:
should 'not' be added in the message? Should it be 'before startup is complete..'